src/Microsoft.AspNetCore.Razor.Language/Legacy/ErrorSink.cs

```diff
@@ -28,7 +28,7 @@ public ErrorSink()
         /// <summary>
         /// Tracks the given <paramref name="error"/>.
         /// </summary>
-        /// <param name="error">The <see cref="RazorError"/> to track.</param>
+        /// <param name="error">The <see cref="RazorDiagnostic"/> to track.</param>
         public void OnError(RazorDiagnostic error) =>_errors.Add(error);
```

Comment:
I'm fine with keeping the method name and parameter name as-is. Right now we don't issue warnings.